### DISCUSSION TITLE
Equipo 9 Proyecto Final

### DISCUSSION THREAD

**Discussion by KasturiSShah** [PatronesHermosos/design-thinking-1ra-generacion#14] [Proyecto Final]
Hola Claudia y Lucero
Aquí está el hilo en el foro para su equipo. Podrían conversar aquí sobre sus avances con el proyecto y avisar el Equipo de Educación de Patrones Hermosos sobre cualquier pregunta que tengan al respecto.
Con la emoción de seguir el desarrollo de su proyecto,
Kasturi

**Comment by KasturiSShah** [comment]
Hola Claudia y Lucero, buenas tardes,
Su compañera Mariela me avisó que quieren pedir por una prórroga, claro que sí, no hay ningún problema. Ahora tienen prórroga hasta este viernes 10 de marzo.
Saludos,
Kasturi

**Comment by KasturiSShah** [comment]
Hola @ClauTorr y Lucero

Gracias por entregar la etapa 0 del proyecto final. He revisado la entrega y escribo con retroalimentación. En primer lugar, me parece que eligieron un tema importante que afecta a madres en todas partes del mundo. Me alegre que están dedicando el proyecto a la pregunta importante de oportunidades laborales pare madres que quieren re-entrar el mundo laboral.

Mirando al protocolo, tengo varias sugerencias: sería una buena idea empezar con algo para romper el hielo (una broma, una pregunta, una historia, lo que sea) y luego pasar a la pregunta 1. Tengo una preocupación sobre la pregunta 5: si la entrevistada/o no ha tenido un proceso de selección, tendré que decir no y en este caso, ¿qué aprenderán? Una opción sería preguntar específicamente sobre el proceso de buscar trabajo que están haciendo (por ejemplo, ¿cómo están buscando oportunidades -- en LinkedIn, Indeed, etc? ¿qué recursos están usando para sus CVs / cartas / etc?).

Otra sugerencia sería agendar la entrevista lo antes posible para poder hacerla y organizar la información recogida en mapas (de afinidad o de empatía o de viaje).

¿Tienen preguntas para mí? Cualquier consulta, estoy a la orden.

Mucha suerte y adelante,
Kasturi

**Reply by KasturiSShah** [reply]
P.D. Para la etapa 1, estoy dando a ustedes una prórroga hasta este domingo. Me avisan cómo andan con esta etapa, para poder retroalimentar la etapa 1 también.

**Comment by LuceroCondoriRivera** [comment]
Hola, gracias.
Si estamos organizando las ideas. Nos extendimos mucho en el tema. Pero ya vamos tomando el rumbo.
Mas bien para poder editar el GitHUB. Como pido permiso?

**Reply by KasturiSShah** [reply]
Hola @LuceroCondoriRivera 
He enviado una invitación a tu cuenta Github. Recibirás la invitación por correo. **Necesitas aceptar la invitación** para poder colaborar con nosotras en Github, editar los archivos, etcetera. Me avisas como andas con este proceso.
Me alegre que están organizando las ideas. Cuando hayan entregado la etapa 1 a Github, me avisen, para poder revisar y dar retroalimentación.
Saludos,
Kasturi

**Comment by LuceroCondoriRivera** [comment]
Muchas gracias,
Ya lo hice.
Saludos
---------------------------------------------------------------------
Lucero Condori Rivera

**Comment by KasturiSShah** [comment]
Hola @ClauTorr @LuceroCondoriRivera 

Espero estén bien y que descansaron durante el feriado.

Escribo con seguimiento sobre su proyecto final. ¿Cómo andan con las etapas  1, 2 y 3? Hay suficiente tiempo para ponerse al corriente con las etapas. El propósito principal del proyecto final es aplicar los conceptos en los módulos en un proyecto. Las etapas son cortas, y no es importante ni necesario entregar etapas pulidas. Lo importante es aplicar los conceptos y aprender por hacer las técnicas en vivo. Animo a ustedes a completar las etapas especialmente durante el curso cuando tengan oportunidad de discutir ideas con el Equipo Patrones Hermosos durante las sesiones y/o en el foro. Si pudiera apoyar a ustedes en algo, me avisarían.

Saludos a la distancia,
Kasturi

**Comment by LuceroCondoriRivera** [comment]
Buenas noches,
Tuvimos contratiempos por el tema de salud de mi compañera. Y por desfases
en los tiempos.
Pero ya retomamos actividades, y estamos completando. ¿Hasta qué fecha
contamos para poder completarlo?
disculpe las molestias y gracias.
Saludos
---------------------------------------------------------------------
Lucero Condori Rivera

**Reply by KasturiSShah** [reply]
Hola @LuceroCondoriRivera y @ClauTorr 

Gracias por escribirme.  @ClauTorr espero te encuentres mucho mejor, que recuperas pronto!

Para contestar a la pregunta, hay dos fechas límites importantes:
- la clausura del curso en la cual darán presentación de su proyecto. Su equipo está asignado al día **miércoles 26 de abril** para presentar (horario disponible [aquí](https://docs.google.com/spreadsheets/d/1GDWqDjb5OVqz2vWRM_DoO0wrFM4o14gEICREDq8ZxQ4/edit?usp=sharing)). Déjame saber si les conviene este día o si prefieren cambiar su día asignado al día sábado. El día de la presentación, no es necesario tener un proyecto completado ni pulido. Sería excelente simplemente mostrar los avances a los demás alumnas y el Equipo de Educación presentes, y discutir las cosas que habrían desarrollado más si hubieran tenido más tiempo.
- La fecha límite para la etapa 6 (es decir, la última etapa) del proyecto será el domingo próximo: **30 de abril de 2023**. Si necesitan una prórroga, no habrá ningun problema. Déjenme saber.

Si tuvieran tiempo para trabajar más, mi sugerencia sería:

1. hacer una entrevista con un usuario
2. usar esta conversación cómo base para crear un prototipo low fidelity (es decir, un dibujo del proyecto)
3. mostrar este prototipo al usuario (puede ser el mismo usuario con que platicaron anteriormente)
4. actualizar el prototipo según la información recibida desde el usuario
5. crear un plan para el futuro usando esa plantilla Sprint que preparamos para este curso [aquí](https://docs.google.com/spreadsheets/d/1aboqGYEJvPv77T5M_uEyUy83oAVKXS2J1Jxi2_48vLQ/copy?usp=sharing)

Podrían avanzar cómo puedan con el tiempo disponible, no es nada necesario completar todo antes de la presentación. Ya que hay dos personas en este equipo, podrían dividir tareas, por ejemplo: una persona haciendo puntos 1 y 3, otra persona haciendo puntos 2 y 4. El punto 5 no necesita que los otros puntos sean completados, así que pueden empezar ya con el punto 5.

Cualquier consulta, aquí ando. Si quisieran retroalimentación después de haber completado cualquier etapa, no duden en avisarme por este medio.

Mucha suerte, mucho éxito,
Kasturi

**Comment by LuceroCondoriRivera** [comment]
Muchas gracias.

---------------------------------------------------------------------
Lucero Condori Rivera

**Comment by LuceroCondoriRivera** [comment]
Buenas noches,
Ya se ha completado las actividades hasta la etapa 3 con el prototipo low
fidelity. Por favor podrían cambiar la fecha de presentación y a qué hora
sería, muy agradecida por el espacio.
Saludos cordiales.
---------------------------------------------------------------------
Lucero Condori Rivera

**Reply by KasturiSShah** [reply]
Hola @LuceroCondoriRivera 

Gracias por escribirnos. Justo te iba a escribir preguntando si quieres presentar el sábado. 
Ahora estás agendada a presentar el sábado después del equipo 5 (horario [aquí](https://docs.google.com/spreadsheets/d/1GDWqDjb5OVqz2vWRM_DoO0wrFM4o14gEICREDq8ZxQ4/edit?usp=sharing)).

Revisé el prototipo low fidelity; me parece muy interesante las diagramas y la manera de navegar entre varias partes de la página. ¡Adelante!

Hasta el sábado, me avisas cualquier cosa.
Kasturi
